digitalocean/createblockstorage.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -e
+
+if [ $# -eq 0 ]
+  then
+    echo "usage: bash createblockstorage.sh [droplet-name]" && exit 1
+fi
+
+DROPLET_NAME=$1
+
+#Install doctl
+
+echo "Installing doctl..."
+
+sudo snap install doctl
+
+#Create config folder as suggested in doctl documentation
+
+sudo mkdir -p ~/.config
+
+#Access doctl account
+
+echo "Accessing digital ocean account..."
+doctl auth init --context data-migration
```

Comment:
Btw, this cmd will prompt the user to enter the token. As we spoke before this might be an issue.

I tried to use pipe to automatically do this:

```suggestion
echo $TOKEN | doctl auth init --context data-migration
```

but to no avail. Any suggestions?

Reply:
Hmm, is the text ok? To pass through we would have to find a non interactive mode flag for this command.

Reply:
Yeah, I tried with --acess-token flag but for some reason it doesn't work alongside with --context flag. I'll remove the second one as I don't think it really matters for our purpose here.